lib/grpnames.gd

```diff
@@ -24,8 +24,8 @@
 ##
 ##  <Description>
 ##    For normal subgroups <A>U</A> and <A>V</A> of <A>G</A>,
-##    IsTrivialNormalIntersection returns
-##    true if <A>U</A> and <A>V</A> intersect trivially, and false otherwise.
+##    <Ref Oper="IsTrivialNormalIntersection"/> returns <K>true</K> if
```

Comment:
````StructureDescription```` had this already, but somehow, never together with a link. Maybe because it referenced a function instead of an operation? I am not sure. Maybe @alex-konovalov can shed some light on how to do this properly.